randovania/games/prime2/json_data/Agon Wastes.txt

```diff
@@ -1889,8 +1885,15 @@ Extra - in_dark_aether: True
       Dark Agon Key 1 and Dark Agon Key 2 and Dark Agon Key 3 and Morph Ball Bomb and Morph Ball and Space Jump Boots and Bomb Space Jump (Expert) and Single Room Out of Bounds (Expert) and Slope Jump (Expert) and Standable Terrain (Expert) and Dark World Damage ≥ 175
   > Event - Amorbis
       All of the following:
-          Morph Ball Bomb and Morph Ball
-          Space Jump Boots or Bomb Jump (Beginner)
+          Morph Ball Bomb and Morph Ball and Dark World Damage ≥ 50 and Activate Safe Zone
```

Comment:
Activating the safe zone should be moved outside of this, because this currently forces either power, light, or annihilator to be valid. Starting with only dark beam makes this path invalid. I would group it as:

All of the following: Morph, Bombs, Dark World Damage > 50 and (Activate zone or damage 30)

Reply:
Understood. Will update.